src/modules/PositionModule.cpp

```diff
@@ -315,6 +316,13 @@ void PositionModule::sendOurPosition()
     // If we changed channels, ask everyone else for their latest info
     LOG_INFO("Send pos@%x:6 to mesh (wantReplies=%d)", localPosition.timestamp, requestReplies);
     sendOurPosition(NODENUM_BROADCAST, requestReplies);
+
+#ifdef USERPREFS_CONFIG_DISCOVERY_CHANNEL_HASH
+    // If the user wants discovery on a different channel, send only nodeinfo to that channel
+    // sendOurNodeInfo() will automatically handle routing a nodeinfo packet to the correct channel
+    assert(nodeInfoModule);
+    nodeInfoModule->sendOurNodeInfo();
```

Comment:
Please remove this part. We shouldn't just send out a NodeInfo **every** time immediately after you're sending out a position.

Reply:
That's fair. What are your thoughts on this -> I think many people use the smart broadcast mechanism to do wardriving node discovery, or they use it because its a reliable way to pickup nearby nodes as you move around (ie. send a packet when my position changes sufficiently, receive responses). What would be a way to maintain this in principle, but address your concern of sending it every time. 

Would 1 out of every 4 position sends work?

Reply:
How is wardriving with the right primary channel Greek to them when they already set up a private primary channel for precise location sharing, and LongFast as secondary to communicate with the public? 
You’re trying to achieve parallel communication channels using the same physical medium. If you want to educate users, you should educate them that you can’t just duplicate traffic on a very low bandwidth mesh.

Reply:
Done